lib/src/directory_watcher/polling.dart

```diff
@@ -70,7 +70,7 @@ class _PollingDirectoryWatcher
   /// queue exists to let each of those proceed at their own rate. The lister
   /// will enqueue files as quickly as it can. Meanwhile, files are dequeued
   /// and processed sequentially.
-  AsyncQueue<String> _filesToProcess;
+  late AsyncQueue<String?> _filesToProcess;
```

Comment:
Can we make this `late final` and move the initializing expression here?

Reply:
Making it `final` seems good to me, but the initializer is quite complex, so I'm not sure.
Is it a pattern that is considered good?

Reply:
It's a pattern we've started using now that it's available. Looks like the initializer is ~4 lines - I think that's not too bad.

It's not too unusual to have long initializing expressions for top-level `final` variables (up to having immediately invoked function literals as the initializing expression), and a class level `late final` now fits a similar space.

Reply:
OK, thank you for the explanation.
I updated the code.